src/CssManager.php

```diff
@@ -950,7 +950,7 @@ function setCSSboxshadow($v)
 					}
 				}
 			}
-			if (!isset($new['col'])) {
+			if (!empty($new['col'])) {
```

Comment:
It should be `empty`, not `!empty`. And what about other `isset` occurences in your first commit? The same should be applied to them as well (sorry if it weren't clear from the line-comment).